ci-operator/config/openshift/microshift/openshift-microshift-release-4.20.yaml

```diff
@@ -228,6 +228,38 @@ tests:
       MICROSHIFT_OS: rhel-9.6
       SUITE: ai-model-serving
     workflow: openshift-microshift-e2e-bare-metal-tests
+- as: e2e-aws-tests-release
+  run_if_changed: ^(test/scenarios-bootc/releases/.*|test/scenarios/releases/.*)$
+  steps:
+    cluster_profile: aws-2
+    env:
+      EC2_INSTANCE_TYPE: c5.metal
```

Comment:
Do we need the extra 24 CPUs from c5 in respect to c5n? Asking because the c5n is a bit cheaper.

Reply:
`c5n.metal` instance has 72 CPUs and [there are 13 bootc releases scenarios](https://github.com/openshift/microshift/tree/main/test/scenarios-bootc/releases). As I can see in the code every VM is created with 4 CPUs. So, with `c5n.metal` will only be room left for 72-13*4=20 CPUs or 5 VMs.

QE team is planning to add more scenarios in the following days, this is just an initial set to validate it's working.

So, I'd prefer to stick with `c5.metal` instances (96 CPUs).

Reply:
If thats happening soon then lets stick to c5. Do you have any special requirements on network? c5n is faster but I dont think we are maximizing that.